komfydb/execution/aggregator.h

```diff
@@ -0,0 +1,60 @@
+#ifndef __AGGREGATOR_H__
+#define __AGGREGATOR_H__
+
+#include <string>
+
+#include "absl/container/flat_hash_map.h"
+#include "absl/status/statusor.h"
+
+namespace komfydb::execution {
+
+class Aggregator {
+ public:
+  static const inline std::vector<int> NO_GROUPING = {};
+  enum AggregateType {
```

Comment:
Please add a comment what does exactly `NONE` represent as I think it's quite tricky. It should be in  `// XXX <description>` format.